diff -urN midgard-lib-1.4.1_5/configure midgard-lib-1.4.1_5.grzegorz/configure
--- midgard-lib-1.4.1_5/configure	Wed Jun 20 12:36:01 2001
+++ midgard-lib-1.4.1_5.grzegorz/configure	Sat Apr 14 16:43:31 2001
@@ -2690,7 +2690,7 @@
 echo "$ac_t""$REPLIGARD_OWNER" 1>&6
 
 REPLIGARD_OWNER_RESULT=yes
-/usr/bin/id $REPLIGARD_OWNER >/dev/null 2>/dev/null || REPLIGARD_OWNER_RESULT=no
+/bin/id $REPLIGARD_OWNER >/dev/null 2>/dev/null || REPLIGARD_OWNER_RESULT=no
 if test "$REPLIGARD_OWNER_RESULT" = "no" ; then
     { echo "configure: error: Could not find user '$REPLIGARD_OWNER' on system to make it owner of Repligard
 binary. Make sure that you've used --with-repligard-owner option with 
